src/viewer/scene/mesh/Mesh.js

```diff
@@ -212,6 +212,7 @@ class Mesh extends Component {
      * @param {Boolean} [cfg.edges=false] Indicates if the Mesh's edges are initially emphasized.
      * @param {Number[]} [cfg.colorize=[1.0,1.0,1.0]] Mesh's initial RGB colorize color, multiplies by the rendered fragment colors.
      * @param {Number} [cfg.opacity=1.0] Mesh's initial opacity factor, multiplies by the rendered fragment alpha.
+     * @param {Boolean} [cfg.background=false] Indicates if the Mesh is the background, such as skybox.
```

Comment:
Move two lines upper.

Reply:
Indicates if the Mesh should act as background, e.g., it can be used for a skybox.